lib/tasks/fed_email_domains.rake

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+require 'csv'
+
+# rubocop:disable Rails/SkipsModelValidations
+namespace :fed_email_domains do
+  task :load, %i[s3_secrets_path] => [:environment] do |_task, args|
+    # Need to increase statement timeout since command takes a long time.
+    ActiveRecord::Base.connection.execute 'SET statement_timeout = 200000'
+    file = Identity::Hostdata.secrets_s3.read_file(args[:s3_secrets_path])
+    names = file.split("\n")
+    FedEmailDomain.insert_all(names.map { |name| { name: } })
```

Comment:
now that we have the `FedEmailDownloader` class, what if this rake task called that, and all this logic was moved into that file?

Reply:
Im fine with that as well, wasnt sure if we wanted to have it there or not, im fine with both.